Algorithms/Sequences/DivisorsCountSequence.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+using System.Numerics;
+
+namespace Algorithms.Sequences
+{
+    /// <summary>
+    ///     <para>
+    ///         Sequence of the number of divisors of n, starting with 1.
+    ///     </para>
+    ///     <para>
+    ///         OEIS: https://oeis.org/A000005.
+    ///     </para>
+    /// </summary>
+    public class DivisorsCountSequence : ISequence
+    {
+        /// <summary>
+        ///     Gets sequence of number of divisors for n, starting at 1.
+        /// </summary>
+        public IEnumerable<BigInteger> Sequence
+        {
+            get
+            {
+                var n = new BigInteger(1);
+                while (true)
+                {
+                    var count = 0;
+                    for (var k = 0; k <= n; k++)
+                    {
+                        BigInteger numerator = k * k + n * k + k;
+                        BigInteger divisor = k + 1;
+                        BigInteger.DivRem(numerator, divisor, out var remainder);
+                        if (remainder == 0)
+                        {
+                            count++;
+                        }
+                    }
+
+                    n++;
+                    yield return count;
+                }
```

Comment:
How does it work? Why not as below?
```suggestion
                yield return BigInteger.One;
                for (var n = new BigInteger(2); ; n++)
                {
                    var count = 2;
                    for (var k = 2; k < n; k++)
                    {
                        BigInteger.DivRem(n, k, out var remainder);
                        if (remainder == 0)
                        {
                            count++;
                        }
                    }

                    yield return count;
                }
```

Reply:
Hm.  I see your point.  Yours is cleaner.

I went looking through the solutions in OEIS, and the implementation I used was the one that looked best at the end of the evening.  I think I'll sleep on my A000008 before submitting that one.